website/docusaurus.config.js

```diff
@@ -3,11 +3,17 @@ module.exports = {
 	tagline: 'Powerfully simple build configuration',
 	url: 'https://premake.github.io/',
 	baseUrl: '/',
+	scripts: [
+		'https://use.fontawesome.com/dd1c9cd9ff.js'
+	],
```

Comment:
We could use [React Font Awesome](https://fontawesome.com/how-to-use/on-the-web/using-with/react) and `import` necessary icons in pages that need them. I'd leave it as it is for now. It's a place for improvement.

Reply:
I started with RFA actually, but found it easier to just work with the class names. And there really isn't other way to do it for the navbar, as far as I could tell. You can only list class names there.

Reply:
Oh, you're right about the navbar. It's ok then 👍